observations/obs_converters/even_sphere/even_sphere.m

```diff
@@ -16,37 +19,56 @@
 diagnostic_test = false;
 
 % Input is in hectopascals - this was intended to be the mandatory levels (see below).
-levels = [1000 850 700 500 400 300 200 150 100 50 30 20 10 7 5];
+% levels = [1000 850 700 500 400 300 200 150 100 50 30 20 10 7 5];
 
 % the nice thing about standards is there are so many of them.  mandatory levels, according to:
 % wisconsin: http://www.meteor.wisc.edu/~hopkins/aos100/raobdoc.htm
-% the surface, 1000, 925, 850, 700, 500, 400, 300, 250, 200, 150, 100, 70, 50, and 10 mb.
+% the surface [1000 925 850 700 500 400 300 250 200 150 100 70 50 10] mb.
 %
 % the AMS glossary: http://amsglossary.allenpress.com/glossary/search?id=mandatory-level1
-% 1000, 850, 700, 500, 400, 300, 200, 150, 100, 50, 30, 20, 10, 7, 5, 3, 2, and 1 mb.
+% [1000 850 700 500 400 300 200 150 100 50 30 20 10 7 5 3 2 1] mb.
 %
 % the Office of Federal Coordinator of Meterology: http://www.ofcm.gov/fmh3/text/chapter5.htm
-% 1000, 925, 850, 700, 500, 400, 300, 250, 200, 150, 100, 70, 50, 30, 20, 10 hPa, plus it says
-% that these levels should be considered standard: 7, 5, 3, 2, and 1 hPa.
+% [1000 925 850 700 500 400 300 250 200 150 100 70 50 30 20 10] hPa 
+% plus it says that these levels should be considered standard: 7, 5, 3, 2, and 1 hPa.
 %
 % the ones that this program generates right now:
-% 1000, 850, 700, 500, 400, 300, 200, 150, 100, 50, 30, 20, 10, 7, 5 mb.
+% Zagar OSSE; we've set   no_obs_assim_above_level = 5, which is slightly higher than 36 hPa in CAM6.
+levels     = [1000  925  850  700  500  400  300  250   200  150  100  70   50   40 ]
+% From NCEP+ACARS/201812_6H/obs_seq2018120100
+% They could (should?) come from obs_converters/obs_error/ncep_obs_err_mod.f90.
+T_variance = [1.44  1.0  0.64 0.64 0.64 0.64 0.81 1.44  1.44 1.0  0.64 0.64 0.81 1.0]
+W_variance = [1.96  2.25 2.25 2.56 4.41 6.76 9.0  10.24 7.29 5.76 4.41 4.41 4.41 4.41]
 
 % Date information is overwritten by create_fixed_network_sequence
-year = 2008;
-month = 10;
-day = 1;
+year = 2017;
+month = 12;
+day = 25;
 hour = 0;
 
+% Model grid for comparison on the obs distribution plot
+% T85   256x128 points on A-grid
```

Comment:
these comments are very cam specific.  they don't belong here in a generic script.  i'd put comments like these in the cam version of some scripts, or in a readme.  this won't help anyone running mpas, for example.